mkgu_packaging/dicarlo/marques/marques_devalois1982a.py

```diff
@@ -0,0 +1,46 @@
+
+import numpy as np
+import pandas as pd
+from marques_utils import gen_sample
+from brainio_collection.packaging import package_data_assembly
+from brainio_base.assemblies import DataAssembly
+
+DATA_DIR = '/braintree/data2/active/users/tmarques/bs_datasets/or_po_DeValois1982.csv'
+ASSEMBLY_NAME = 'dicarlo.Marques_devalois1982a'
```

Comment:
in terms of naming, I believe this assembly should be called `devalois.DeValois1982` following the current convention (`<PI lastname>.<first author last name><year>`)

Reply:
I treated the assembly as being generated by me since this was not data that was contributed to us by someone else but was dug out of papers from me. In that case the dicarlo.Marques would obey the current convention. The _devalois1982a is a reference to the paper from where the data came from. I can change it if you think it's more appropriate the other way.

Reply:
Changed to devalois.DeValois1982a

Reply:
I think the `Marques2020` should come into play for the benchmark. But we can certainly discuss adding an additional reference for the data too, I would also like to give credit to the people who dig out the data

Reply:
do you think the `a` at the end is necessary? Are you using it because there are two 1982 papers and this is the first? Your call

Reply:
Back in the old days, these very descriptive papers would be published back-to-back with other studies. In many cases there would be 3 or more studies of the same authors in the same journal. This is one of those cases. The most recent commit has a data assembly from the first study (paper a). There is another assembly which (schiller.Schiller1976c) that I don't use the first 2 papers but nevertheless left the c to specify which of the studies the paper is.